tools/lib.sh

```diff
@@ -75,3 +75,7 @@ function contains_element() {
   return 1
 }
 
+function validate_config() {
+  log "Validating json config file $1"
+  python -m json.tool $1 1>/dev/null 
```

Comment:
You should use `osqueryd --config_path <path> --config_check` to check the config since boost's config parser parses a superset of normal JSON.